docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,317 @@
+# Shapella mainnet parameters
+
+## Mainnet addresses
+
+```python
+lidoLocator = "0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb"
+```
+
+## NodeOperatorsRegistry
+
+```python
+stuckPenaltyDelay = 432000  # 5 days as seconds
+```
+
+## OracleDaemonConfig
+
+```python
+NORMALIZED_CL_REWARD_PER_EPOCH=64
+NORMALIZED_CL_REWARD_MISTAKE_RATE_BP=1000
+REBASE_CHECK_NEAREST_EPOCH_DISTANCE=1
+REBASE_CHECK_DISTANT_EPOCH_DISTANCE=23  # 10% of AO 255 epochs frame
+VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS=7200 # 1 day
+VALIDATOR_DELINQUENT_TIMEOUT_IN_SLOTS=28800 # 4 days
+PREDICTION_DURATION_IN_SLOTS=50400
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
```

Comment:
Constant approved - in line with the proposed design on "bunker mode" with accepted removing of +1 day edge case with bunker triggering due to last epoch negative rebase